tests/suites/test_suite_psa_crypto_se_driver_hal_mocks.function

```diff
@@ -53,12 +63,13 @@ static struct
 
 static void mock_teardown( void )
 {
+    memset( &mock_generate_data, 0, sizeof( mock_generate_data ) );
     memset( &mock_import_data, 0, sizeof( mock_import_data ) );
     memset( &mock_export_data, 0, sizeof( mock_export_data ) );
     memset( &mock_export_public_data, 0, sizeof( mock_export_public_data ) );
+    memset( &mock_sign_data, 0, sizeof( mock_sign_data ) );
     memset( &mock_allocate_data, 0, sizeof( mock_allocate_data ) );
     memset( &mock_destroy_data, 0, sizeof( mock_destroy_data ) );
-    memset( &mock_generate_data, 0, sizeof( mock_generate_data ) );
```

Comment:
In commit " crypto_se_driver: add signing mock test", don't move the location of mock_generate_data memsetting.

Reply:
Fixed